Makefile

```diff
@@ -45,7 +45,18 @@ PUSH_TARGETS := $(patsubst %, push-%, $(DISTRIBUTIONS))
 BUILD_TARGETS := $(patsubst %, build-%, $(DISTRIBUTIONS))
 TEST_TARGETS := $(patsubst %, build-%, $(DISTRIBUTIONS))
 
-.PHONY: $(DISTRIBUTIONS) $(PUSH_TARGETS) $(BUILD_TARGETS) $(TEST_TARGETS)
+build:
```

Comment:
Why not also add a `test` rule here to address #4

Reply:
no test files

Reply:
OK, but that doesn't mean that we can't add the make target so that when we *do* add tests these will be triggered. If #4 is about actually adding unit tests, please update the issue description to call that out.

Reply:
I get the idea, an empty `make test`, makes sense